nvdaHelper/vbufBackends/mshtml/node.cpp

```diff
@@ -292,18 +292,39 @@ class CHTMLChangeSink : public IHTMLChangeSink {
 
 };
 
-MshtmlVBufStorage_controlFieldNode_t::MshtmlVBufStorage_controlFieldNode_t(int docHandle, int ID, bool isBlock, MshtmlVBufBackend_t* backend, bool isRootNode, IHTMLDOMNode* pHTMLDOMNode,const wstring& lang): VBufStorage_controlFieldNode_t(docHandle,ID,isBlock), language(lang) {
+MshtmlVBufStorage_controlFieldNode_t::MshtmlVBufStorage_controlFieldNode_t(
+	int docHandle,
+	int ID,
+	bool isBlock,
+	MshtmlVBufBackend_t* backend,
+	bool isRootNode,
+	IHTMLDOMNode*
+	pHTMLDOMNode,
+	const wstring& lang
+)
+	: VBufStorage_controlFieldNode_t(docHandle, ID, isBlock)
+	, backend(backend)
+	, pHTMLDOMNode(nullptr)

```

Comment:
Surely if this and other instance variables now have initializers in the h file, it is unnecessary to include them specifically here? Or if I'm wrong, then the other way round... why must we give initial vlaues for these instance variables in the h file if they are never used?

Reply:
That's what I thought, though the analyzer was still complaining, [looking this up](https://en.cppreference.com/w/cpp/language/data_members):
>2) Through a default member initializer, which is a brace or equals initializer included in the member declaration and is used if the member is omitted from the member initializer list of a constructor.

I'll try this again, perhaps something else went wrong with the analyzer that I didn't notice.